test/integration-test/sql-parser/src/test/resources/env/it-env.properties

```diff
@@ -17,3 +17,5 @@
 
 sql.parser.it.enabled=false
 sql.parser.it.report.path=/tmp/
+# sql.parser.it.loader=github,gitee,local
+sql.parser.it.loader=github
```

Comment:
as I know, in the following refactor process, you will apply the strategy pattern. but from this config file, I realized this may not be a strategy pattern .
please think about it.